packages/html-program-viewer/tsconfig.build.json

```diff
@@ -4,5 +4,5 @@
     "rootDir": "src"
   },
   "include": ["src"],
-  "exclude": ["**/*.test.ts", "**/*.test.tsx"]
+  "exclude": ["src/index.ts", "src/emitter.ts", "**/*.test.ts", "**/*.test.tsx"]
```

Comment:
i have no idea why this is needed now, but basically it was failing the react build because it used some file system apis